src/outputstyle.rs

```diff
@@ -46,3 +46,27 @@ impl Default for OutputStyle {
         }
     }
 }
+
+pub struct TestCaseStyle {
+    pub success: Style,
+    pub failure: Style,
+    pub error: Style,
+    pub title: Style,
+    pub stderr: Style,
+    pub out: Style,
+    pub whitespace: Option<Style>,
+}
+
```

Comment:
I think these could be part of `OutputStyle` instead of their own separate struct (no need to define title and whitespace styles in two different places)